tests/Doctrine/Tests/DBAL/Schema/ColumnTest.php

```diff
@@ -55,6 +56,15 @@ public function testToArray()
         self::assertEquals($expected, $this->createColumn()->toArray());
     }
 
+    /**
+     * @group legacy
+     * @expectedDeprecation The "unknown_option" option is not supported, setting it is deprecated and will cause an exception in 3.0
```

Comment:
TIL there's `@expectedDeprecation`. Thanks!

Reply:
You're very welcome!